src/cnxml/serialize.js

```diff
@@ -148,6 +150,16 @@ function xref(obj, children) {
 }
 
 
+/**
+ * Serializer for code.
+ */
+function code(obj, children) {
+    return <code display="block">
+        {obj.getTexts().map(t => t.text)}
+    </code>
```

Comment:
Why are you manually serializing contents rather than using `children`?

Reply:
`"element.props.children is undefined"`